backend/game/scribble/handler/gamemaster/gamemaster.go

```diff
@@ -300,10 +312,14 @@ func (h *Handler) run() {
 		case ShowResults:
 			h.showResults()
 			h.gameState = EndGame
+			fallthrough
+		case EndGame:
 			break
```

Comment:
Can this just have the logic below, i.e. put the `h.changeGameStateTo(Ended)` here instead of down there?

Reply:
no, i need to break out of the for loop